frontend/cypress/.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules/
+node/
+
+!package.json
+!package-lock.json
```

Comment:
Why is there a need to un-ignore the `package.json` files here? They should be automatically installed when we run `installCypress` right?

Reply:
Hmm. Only `!package-lock.json` is needed.

`package-lock.json` is auto-generated but best practices advise to commit it. As it helps to speed up CIs. 

https://docs.npmjs.com/cli/ci.html

Reply:
>package-lock.json` is auto-generated but best practices advise to commit it. As it helps to speed up CIs.

Hmmm, guess maybe we can also commit the `package.json` and `package-lock.json` in the root directory as well, but can leave that for a separate PR/issue.

Reply:
If you wanna use package.json need to use nvm to fix npm and node version if not they will do all the dependency checking anyways.